docker-compose-demo.yml

```diff
@@ -102,7 +102,28 @@ services:
     environment:
       - SERVER_JWTPUBKEYURL=https://oidc:8080/jwk
     volumes:
-    - cacert:/etc/ssl/certs/
+      - cacert:/etc/ssl/certs/
+
+
+  reencrypt:
+    image: ghcr.io/neicnordic/sensitive-data-archive:v0.3.25
+    command: [sda-reencrypt]
+    container_name: reencrypt
+    depends_on:
+      credentials:
+        condition: service_completed_successfully
+    ports:
+      - "50443:50443"
```

Comment:
```suggestion
```
This shouldn't be needed

Reply:
Right, I mapped the port for testing the reencrypt service directly in the CLI and it's not needed when `download` is in the same network as `reencrypt`.

Reply:
Thanks! Fixed in c20bfe0